src/applications/personalization/connected-accounts/components/ConnectedApp.jsx

```diff
@@ -36,75 +36,68 @@ class ConnectedApp extends React.Component {
   };
 
   render() {
-    const { href, logo, title, created, grants } = this.props.attributes;
+    const { logo, title, created, grants } = this.props.attributes;
     const cssPrefix = 'va-connected-acct';
     const toggled = this.state.detailsOpen
       ? `${cssPrefix}-details-toggled`
       : '';
     const lastClass = this.props.isLast ? `${cssPrefix}-last-row` : '';
     return (
-      <tr>
-        <table className={`${cssPrefix}-row-table ${lastClass}`}>
-          <tbody>
-            <tr className={`${cssPrefix}-row ${toggled}`}>
-              <th scope="row">
-                <a href={href} className="no-external-icon">
-                  <img src={logo} alt={`${title} logo`} width="100" />
-                </a>
-              </th>
-              <th>
-                Connected on {moment(created).format('MMMM D, YYYY h:mm A')}
-              </th>
-              <th className={`${cssPrefix}-row-details`}>
-                <button
-                  className={`${cssPrefix}-row-details-toggle va-button-link`}
-                  aria-expanded={this.state.detailsOpen ? 'true' : 'false'}
-                  onClick={this.toggleDetails}
-                >
-                  Details
-                  <i
-                    className={`fa fa-chevron-${
-                      this.state.detailsOpen ? 'up' : 'down'
-                    }`}
-                  />
-                </button>
-                <AccountModal
-                  appName={title}
-                  modalOpen={this.state.modalOpen}
-                  onCloseModal={this.closeModal}
-                  onConfirmDelete={this.confirmDelete}
-                />
-              </th>
-            </tr>
-            {this.state.detailsOpen && (
-              <tr className={`${cssPrefix}-row-details-block`}>
-                <th colSpan="3">
-                  <div className={`${cssPrefix}-row-details-block-wrapper`}>
-                    <div className={`${cssPrefix}-row-details-block-content`}>
-                      <p>
-                        <a href={href}>{title}</a>
-                        &nbsp;can view your:
-                        <button
-                          aria-label={`Disconnect ${title} from your account`}
-                          className="usa-button-primary"
-                          onClick={this.openModal}
-                        >
-                          Disconnect
-                        </button>
-                      </p>
-                      <ul>
-                        {grants.map((a, idx) => (
-                          <li key={idx}>{a.title}</li>
-                        ))}
-                      </ul>
-                    </div>
-                  </div>
-                </th>
-              </tr>
-            )}
-          </tbody>
-        </table>
-      </tr>
+      <div
+        className={`${cssPrefix}-row ${toggled} ${lastClass}`}
+        onClick={this.toggleDetails}
```

Comment:
Is this definitely an issue for screen readers? The way I'm thinking of it, this is a small enhancement for people who are navigating with a mouse and/or their finger on a touch screen.

Since the div _isn't_ an interactive element and doesn't have a tab index, it won't be highlighted if you're tabbing through, or by screen readers, so it shouldn't cause a problem for them? Am I misunderstanding?

I don't have a strong objection to removing it, but I would prefer to keep it if it's not problematic.

Reply:
@kalilsn Good point, and one I hadn't considered RE mobile users. I'm on board with keeping it, and we can revisit if the 508 office flags it for review.